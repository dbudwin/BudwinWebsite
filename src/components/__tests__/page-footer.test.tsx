import "@testing-library/jest-dom";

import PageFooter from "../page-footer";
import React from "react";
import renderer from "react-test-renderer";

it("shows the footer with copyright message and personal links", () => {
    const footer = renderer.create(<PageFooter copyrightYear={2020} />).toJSON();

    expect(footer).toMatchSnapshot();
});
