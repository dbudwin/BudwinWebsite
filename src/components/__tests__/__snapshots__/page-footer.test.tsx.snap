// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shows the footer with copyright message, personal links, and dad joke 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content"
      style={
        {
          "textAlign": "center",
        }
      }
    >
      © 
      2020
       
      <a
        href="https://www.budw.in/"
        rel="noopener noreferrer"
        target="_blank"
      >
        budw.in
      </a>
       - Hosted with ❤️ on 
      <a
        href="https://bit.ly/dbudwin-digitalocean"
        rel="noopener noreferrer"
        target="_blank"
      >
        DigitalOcean
      </a>
      , secured by 
      <a
        href="https://letsencrypt.org/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Let's Encrypt
      </a>
      .
    </div>
    <div
      className="content"
      style={
        {
          "textAlign": "center",
        }
      }
    >
      <a
        href="https://bit.ly/dbudwin-linkedin"
        rel="noopener noreferrer"
        target="_blank"
      >
        LinkedIn
      </a>
       | 
      <a
        href="https://bit.ly/dbudwin-medium"
        rel="noopener noreferrer"
        target="_blank"
      >
        Medium
      </a>
       | 
      <a
        href="https://bit.ly/dbudwin-github"
        rel="noopener noreferrer"
        target="_blank"
      >
        GitHub
      </a>
    </div>
    <div
      className="content"
      style={
        {
          "textAlign": "center",
        }
      }
    >
      What do you call a pile of cats? A Meowtain.
    </div>
  </div>
</footer>
`;
